### PR TITLE
Update cgoflags to support raspbian.

### PR DESCRIPTION
**Pull request by happydig**
support install on raspbian.

### CODE REVIEW

File: cgoflags.go [hunk 1]
```diff
@@ -9,5 +9,7 @@ package hdf5
 // #cgo darwin LDFLAGS: -L/usr/local/lib
 // #cgo linux CFLAGS: -I/usr/local/include, -I/usr/lib/x86_64-linux-gnu/hdf5/serial/include
 // #cgo linux LDFLAGS: -L/usr/local/lib, -L/usr/lib/x86_64-linux-gnu/hdf5/serial/
+// #cgo linux arm CFLAGS: -I/usr/local/include, -I/usr/lib/arm-linux-gnueabihf/hdf5/serial/include/
+// #cgo linux arm LDFLAGS: -L/usr/local/lib, -L/usr/lib/arm-linux-gnueabihf/hdf5/serial/
```
Comment (sbinet) [inline cgoflags.go]: ok... perhaps add `!arm` for the other `linux` directives? or add `amd64`?

Comment (happydig) [reply cgoflags.go]: could we use pkg-config directive?  as:
```
 package hdf5
// #cgo pkg-config: hdf5
// #cgo LDFLAGS: -lhdf5_hl
// #include "hdf5.h"
import "C"
```

Comment (happydig) [reply cgoflags.go]: I create pull request: #43.

Comment (happydig) [reply cgoflags.go]: I prefer "amd64". or could we use pkg-config directive? as:
```
 package hdf5
// #cgo pkg-config: hdf5-serial
// #cgo LDFLAGS: -lhdf5_hl
// #include "hdf5.h"
import "C"
```
This passed on my laptops but failed on Travis. Travis failed ot execute 'pkg-config hdf5-serial --cflags" .

Comment (kortschak) [reply cgoflags.go]: The union of build tag must be 𝑈, so please use `!arm` in conjunction with `arm`. If pkg-config works though, that would be the best.

Comment (hz) [reply cgoflags.go]: The status is:
1, On raspbian stretch, libhdf5-dev provides hdf5.pc.
2, On Ubuntu 18.10, libhdf5-dev proides hdf5.pc.
3, On Mac OS homebrew, hdf5 does not provide hdf5.pc. It is due to hdf5 source code without hdf5.pc.
4, On Ubuntu trusty which travis-ci runs, libhdf5-dev does not provide hdf5.pc.

So I will create a pull request with '!arm' option.

Thanks a lot.

Comment (happydig) [reply cgoflags.go]: The status is:
1, On raspbian stretch, libhdf5-dev provides hdf5.pc.
2, On Ubuntu 18.10, libhdf5-dev proides hdf5.pc.
3, On Mac OS homebrew, hdf5 does not provide hdf5.pc. It is due to hdf5 source code without hdf5.pc.
4, On Ubuntu trusty which travis-ci runs, libhdf5-dev does not provide hdf5.pc.

So I will create a pull request with '!arm' option.

Thanks a lot.

Comment (happydig) [reply cgoflags.go]: I filed pull request #46 . Please review it.

**Pull request closed without merging**